TypeCobol/Compiler/CodeElements/Expressions/StorageArea.cs

```diff
@@ -70,6 +72,15 @@ public override string ToString()
         }
     }
 
+    public class SqlStorageArea : StorageArea
+    {
+        public SqlStorageArea(SymbolReference fullNameSymbolReference) : base()
+        {
+            this.SymbolReference = fullNameSymbolReference;
+        }
+        
+    }
```

Comment:
As discussed with @smedilol and @mayanje, we'll cover this new concept in a dedicated issue. Please revert the changes in this file.